src/sonic-py-common/sonic_py_common/security_cipher.py

```diff
@@ -0,0 +1,120 @@
+'''
+
+A common module for handling the encryption and
+decryption of the feature passkey. It also takes
+care of storying the secure cipher at root 
+protected file system
+
+'''
+
+import subprocess
+import threading
+import syslog
+import os
+from swsscommon.swsscommon import ConfigDBConnector
+
+class security_cipher:
```

Comment:
Can we rename this module to "master key manager" or any other better name to align with other popular NOS?

Reply:
I have chosen this name as this module is mainly responsible for securing the cipher. :-)

Reply:
@nmoray Make sense. However, please rename secuirty_cipher class name to master_key_manager as it is not just securing the cipher and also storing and takes care of lifecycle of it.

Reply:
@nmoray Are you planning to make changes for the above suggestion?